crates/astria-sequencer/src/genesis.rs

```diff
@@ -5,9 +5,20 @@ use serde::{
 };
 
 /// The genesis state for the application.
-#[derive(Debug, Deserialize, Default)]
+#[derive(Debug, Deserialize)]
 pub(crate) struct GenesisState {
     pub(crate) accounts: Vec<Account>,
+    #[serde(deserialize_with = "deserialize_address")]
+    pub(crate) authority_sudo_key: Address,
```

Comment:
Does a 0 address make sense as a default? A hash of `0`s is valid, so an address of `0`s is valid.

Reply:
Alternatively, `Option<Address>`?